### ISSUE TITLE
Update '10X Chromium' to include other single cell technologies

### ISSUE THREAD

**Issue by ustervbo** [airr-community/airr-standards#810]
Currently, the subsection [10X Chromium](https://docs.airr-community.org/en/latest/miairr/metadata_guidelines.html#x-chromium) is focused on 10x, but other manufacturers have entered the single cell arena. Methods like [TIRTL-seq](https://www.biorxiv.org/content/10.1101/2024.09.16.613345v1) may also gain traction.

We should update the section to be more platform independent.

**Comment by bcorrie** [comment]
Yes, this should probably be generalized towards describing single-cell experiments maybe using 10X as an example.

I see that TIRTL-seq uses MiXCR (at least in the paper) does anyone know if their AIRR support includes cell_index for paired chains? I see that MiXCR has an exportAIRR command, but it isn't on the list of "AIRR compliant software" so hasn't passed the certification. We have never used MiXCR to process a study with paired chains, so not sure if a paired chain study in the AIRR format would properly represent paired chains???

Of course 10X is not on the supported software list either, but I know their output is AIRR compliant as we load their data, so this is mostly out of curiosity more than anything else.

**Comment by scharch** [comment]
Like 10X, it's closed source (AFAIK). But the exportAIRR works well enough. Dunno about paired chains.

**Comment by ustervbo** [comment]
TIRTL-seq identified likely combinations of TCRa and b using 384 well
plates with about 25000 cells/well. The indices you have in TIRTL-seq are
for the wells, not the cells.

Other approaches connect TCRa-b and BCR H-L at the single cell level. There
are no identifying indices apart from (maybe) UMIs.
